### PR TITLE
added recipe for lavender latte

### PR DESCRIPTION
**Pull request by Shreya-L**
Added new recipe for #725

### CODE REVIEW

File: src/recipes/lavender-latte.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            "quantity": "1-2",
+            "measure": "tablespoons",
+            "ingredient": "lavender syrup"
+        },
+
+    ],
+    "directions": [
+        "1. Bring water to a boil, add dried lavender (or extract) and sugar",
+        "2. Stir sugar, lavender, water and reduce heat until liquid becomes more of a syrup",
+        "3. Add 1-2 tablespoons of lavender syrup to strong coffee and add milk to enjoy!"
```
Comment (alfg) [inline src/recipes/lavender-latte.json:41]: You can remove the numerations here. This array will be rendered as an ordered list.

File: src/recipes/lavender-latte.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        {
+            "quantity": "2/3",
+            "measure": "cup",
+            "ingredient": "strong brewed coffee"
+        },
+        {
+            "quantity": "1-2",
+            "measure": "tablespoons",
+            "ingredient": "lavender syrup"
+        },
```
Comment (alfg) [inline src/recipes/lavender-latte.json:35]:
```suggestion
        }
```

**Review by alfg — CHANGES_REQUESTED** [review]
2 suggestions. Thanks!

File: src/recipes/lavender-latte.json (hunk 1 above)
Comment (Shreya-L) [reply src/recipes/lavender-latte.json:41]: Ah I see that makes sense! Thank you!__

**Review by Shreya-L — COMMENTED** [review]
Thanks!

**Review by Shreya-L — COMMENTED** [review]
Finished

**Comment by alfg** [comment]
One of my favorite drinks. :) Thank you!

**Comment by Shreya-L** [comment]
It's so good! Thank you!

**Comment by Shreya-L** [comment]
How do we get the hacktoberfest-accepted tag after PR (I am relatively new to all of this I'd have to admit).

**Comment by alfg** [comment]
I believe as long as the PR is merged, then it is accepted. You can check on the hacktoberfest progress website. I also added the label just in case!

**Comment by Shreya-L** [comment]
Awesome, thanks for explaining! and for adding the label for me!

**Comment by alfg** [comment]
You're welcome!

**Pull request merged**